.github/workflows/build.sh

```diff
@@ -247,7 +247,8 @@ case "$(uname)" in
   rm -r test/execute/constant_function_argument
   
   # The C code generated by Dredd may require recent C/C++ support.
-  export DREDD_EXTRA_C_ARGS="/std:c17"
+  # Dredd's prelude includes experimental `stdatomic.h`` header.
```

Comment:
Remove redundant second backtick after `stdatomic.h`.

Nit: add "the" before "experimental".

Reply:
Done